regen-network/regen-web
# feat: change default filters on project list page.

- Add environment variable to configure credit class filters to deselect by default on the project list page
- Update label for filter button

## Description

Closes: https://regennetwork.atlassian.net/browse/APP-94



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. define environment variable `VITE_CREDIT_CLASS_FILTERS_TO_DESELECT` for the marketplace with the credit classes that you want to deselect by default on the project list page, e.g. `VITE_CREDIT_CLASS_FILTERS_TO_DESELECT=C03,C02`
2. visit the project list page
3. open filters side drawer
4. check that the desired credit class filter is deselected by default
5. check a deselected filter to ensure that toggling it still works as expected

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
